nunit.sln

```diff
@@ -1,6 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
+# Visual Studio 14
```

Comment:
Especially since those changes are just comments. I think it only happens
if you change the solution. I had included the DNX projects, then backed
out those changes in a later commit.
On Oct 14, 2015 1:20 PM, "CharliePoole" <notifications@github.com> wrote:

> In nunit.sln
> <https://github.com/nunit/nunit/pull/874#discussion_r42023758>:
>
> > @@ -1,6 +1,8 @@
> >  ﻿
> >  Microsoft Visual Studio Solution File, Format Version 12.00
> > -# Visual Studio 2012
> > +# Visual Studio 14
>
> It always seems strange to me that simply opening a solution in a newer
> version of VS causes changes to be made!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/874/files#r42023758>.
>